src/components/TaskModal.vue

```diff
@@ -115,7 +117,7 @@ export default {
 			return this.$store.getters.getAllTags
 		},
 		calendars() {
-			return this.$store.getters.getTaskCalendarsForCurrentUser
+			return this.$store.getters.getTaskCalendarsForCurrentUser.map(calendar => ({ id: calendar.id }))
```

Comment:
inspect with the Vue dev tools. you will get an array of objects that only have an `id` property, nothing else. you might need to map color and displayname too